src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java

```diff
@@ -81,6 +81,10 @@ private void buildAndSendActionInstruction(
     actionInstruction.setUprn(caze.getAddress().getUprn());
     actionInstruction.setUndeliveredAsAddress(caze.getUndeliveredAsAddressed());
 
+    if (caze.getCaseType().equals("CE")) {
```

Comment:
Would prefer not to have this CE check, I presume we're ok sending SecureEstab == False to field for none CE cases?  I thought this might affect SPGs too?

Reply:
We're raising another card for SPGs